wsdl/ver10/schema/onvif.xsd

```diff
@@ -1807,12 +1807,106 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			</xs:extension>
 		</xs:complexContent>
 	</xs:complexType>
+
+	<xs:simpleType name="FrequencyRange">
+    <xs:restriction base="xs:string">
+        <xs:enumeration value="20Hz-60Hz">
+            <xs:annotation>
+                <xs:documentation>Sub-bass frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="60Hz-250Hz">
+            <xs:annotation>
+                <xs:documentation>Bass frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="250Hz-500Hz">
+            <xs:annotation>
+                <xs:documentation>Low midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="500Hz-2kHz">
+            <xs:annotation>
+                <xs:documentation>Midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="2kHz-4kHz">
+            <xs:annotation>
+                <xs:documentation>Upper midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="4kHz-20kHz">
+            <xs:annotation>
+                <xs:documentation>High frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+    </xs:restriction>
+</xs:simpleType>
+
+	<!--===============================-->
+	<!--          FrequencyGainPair            -->
+	<!--===============================-->
+	<xs:complexType name="FrequencyGainPair">
+    	<xs:sequence>
+        <xs:element name="FrequencyRange" type="xs:string">
+            <xs:annotation>
+                <xs:documentation>Defined in tt:FrequencyRange, e.g., 20Hz-60Hz, 2kHz-4kHz.(Readonly)</xs:documentation>
+            </xs:annotation>
+        </xs:element>
+        <xs:element name="Gain" type="xs:float">
+            <xs:annotation>
+                <xs:documentation>Normalized value (0.0 ~ 1.0).</xs:documentation>
+            </xs:annotation>
+        </xs:element>
+		<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first Vendor then ONVIF -->
+    	</xs:sequence>
+	</xs:complexType>
+
+	<!--===============================-->
+	<!--          EQPreset            -->
+	<!--===============================-->
+	<xs:complexType name="EQPreset">
+		<xs:sequence>
+			<xs:element name="Token" type="tt:ReferenceToken">
+				<xs:annotation>
+					<xs:documentation>Unique identifier for the preset.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Name" type="xs:string">
+				<xs:annotation>
+					<xs:documentation>Preset name. (Readonly)</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ScheduleToken" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Optional schedule token (if supported).</xs:documentation>
```

Comment:
How is schedule to be used? Able to set many presets? 

See also <xs:element name="Configuration" type="tt:EQPreset"> in SetEQPresetConfiguration, to be a list?

Reply:
Now this parameter is removed in latest commit and clarified the issue raised during last F2F meeting in Bangkok.
Please check the latest commit[0059eac](https://github.com/onvif/specs/pull/450/commits/0059eaca1b625201bbcab381263a9348e01e7dec)

Reply:
Agreed, EQPresets now unbounded. resolved